vast.conf

```diff
@@ -19,8 +19,11 @@ system {
   ; The unique ID of this node.
   ;node-id = "node"
 
-  ; List of paths to look for schema files.
-  ;schema-paths = <["$VAST_INSTALL_PREFIX/share/vast/schema"]>
+  ; List of paths to look for schema files in ascending order of priority.
+  ; Note: Automatically prepended with
+  ;  ["<binary_directory>/../share/vast/schema", "/etc/vast/schema"].
+  ; Use --no-default-schema to turn off this mechanism.
```

Comment:
In the context of the configuration file, I think it makes more sense to reference the `system.no-default-schema` option. And considering it is right below, I don't think this needs to be  in here at all.

Reply:
I'd rather keep it. Most people don't read example config files top to botton.

Reply:
Fair enough. I don't have a very strong opinion on this.